tests/snippets/whats_left_to_implement.py

```diff
@@ -1,878 +1,874 @@
+        '__rxor__',
+        '__setattr__',
+        '__sizeof__',
+        '__str__',
+        '__sub__',
+        '__subclasshook__',
+        '__xor__',
+        'add',
+        'clear',
+        'copy',
+        'difference',
+        'difference_update',
+        'discard',
+        'intersection',
+        'intersection_update',
+        'isdisjoint',
+        'issubset',
+        'issuperset',
+        'pop',
+        'remove',
+        'symmetric_difference',
+        'symmetric_difference_update',
+        'union',
+        'update',
+    ]),
+    'str': ('', [
+        '__add__',
+        '__class__',
+        '__contains__',
+        '__delattr__',
+        '__dir__',
+        '__doc__',
+        '__eq__',
+        '__format__',
+        '__ge__',
+        '__getattribute__',
+        '__getitem__',
+        '__getnewargs__',
+        '__gt__',
+        '__hash__',
+        '__init__',
+        '__init_subclass__',
+        '__iter__',
+        '__le__',
+        '__len__',
+        '__lt__',
+        '__mod__',
+        '__mul__',
+        '__ne__',
+        '__new__',
+        '__reduce__',
+        '__reduce_ex__',
+        '__repr__',
+        '__rmod__',
+        '__rmul__',
+        '__setattr__',
+        '__sizeof__',
+        '__str__',
+        '__subclasshook__',
+        'capitalize',
+        'casefold',
+        'center',
+        'count',
+        'encode',
+        'endswith',
+        'expandtabs',
+        'find',
+        'format',
+        'format_map',
+        'index',
+        'isalnum',
+        'isalpha',
+        'isdecimal',
+        'isdigit',
+        'isidentifier',
+        'islower',
+        'isnumeric',
+        'isprintable',
+        'isspace',
+        'istitle',
+        'isupper',
+        'join',
+        'ljust',
+        'lower',
+        'lstrip',
+        'maketrans',
+        'partition',
+        'replace',
+        'rfind',
+        'rindex',
+        'rjust',
+        'rpartition',
+        'rsplit',
+        'rstrip',
+        'split',
+        'splitlines',
+        'startswith',
+        'strip',
+        'swapcase',
+        'title',
+        'translate',
+        'upper',
+        'zfill'
+    ]),
+    'tuple': ((), [
+        '__add__',
+        '__class__',
+        '__contains__',
+        '__delattr__',
+        '__dir__',
+        '__doc__',
+        '__eq__',
+        '__format__',
+        '__ge__',
+        '__getattribute__',
+        '__getitem__',
+        '__getnewargs__',
+        '__gt__',
+        '__hash__',
+        '__init__',
+        '__init_subclass__',
+        '__iter__',
+        '__le__',
+        '__len__',
+        '__lt__',
+        '__mul__',
+        '__ne__',
+        '__new__',
+        '__reduce__',
+        '__reduce_ex__',
+        '__repr__',
+        '__rmul__',
+        '__setattr__',
+        '__sizeof__',
+        '__str__',
+        '__subclasshook__',
+        'count',
+        'index',
+    ]),
+    'object': (object(), [
```

Comment:
Do we require an instance? We could do with the class itself right? So, change `object()` to `object`.

Reply:
For `iter` a couple of attributes were only available on instances, but it's probably fine to give classes for other types.